frontend/package-lock.json

```diff
@@ -5129,6 +5130,13 @@
         "node": ">=0.10.0"
       }
     },
+    "node_modules/string-similarity": {
+      "version": "4.0.4",
+      "resolved": "https://registry.npmjs.org/string-similarity/-/string-similarity-4.0.4.tgz",
+      "integrity": "sha512-/q/8Q4Bl4ZKAPjj8WerIBJWALKkaPRfrvhfF8k/B23i4nzrlRj2/go1m90In7nG/3XDSbOo0+pu6RvCTM9RGMQ==",
+      "deprecated": "Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.",
```

Comment:
seems this package is deprecated, consider alternatives such as [Fuse.js](https://fusejs.io/) or [Levenshtein](https://www.npmjs.com/package/fast-levenshtein).